src/donejs-carousel-plugin.js

```diff
@@ -537,15 +561,14 @@ export const ViewModel = DefineMap.extend({
 	* active slide should have higher z-index value
 	*/
 	fadeToActiveSlide() {
-		let classSelector = this.classSelector;
-
+		
 		// only fade if slides are in carousel, not broken on desktop
 		if (!(this.isDesktop && this.carouselOptions.breakOnDesktop)) {
 			// transition length in ms
 			let transitionLength = 1000;
```

Comment:
should probably make this a `const`

Reply:
using `let` leaves the possibility for that variable to be changed, but we now that within the scope of this function `transitionLength` should never change, so better to use `const`.

But i'm being overly pedantic, it's not a huge deal.

Reply:
cool, I'll change it - pedantic is good when it comes to code